DigitalOffice/Services/FileService/Startup.cs

```diff
@@ -0,0 +1,57 @@
+using FileService.Database;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace FileService
+{
+    public class Startup
+    {
+        public IConfiguration Configuration { get; }
+        
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+        
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddDbContext<FileServiceDbContext>(options =>
+            {
+                options.UseSqlServer(Configuration.GetConnectionString("SQLConnectionString"));
+            });
+
+            services.AddControllers();
+        }
+        
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            UpdateDatabase(app);
+
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+
+            app.UseHttpsRedirection();
+            app.UseRouting();
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapControllers();
+            });
+        }
+
+        private static void UpdateDatabase(IApplicationBuilder app)
```

Comment:
why static?

Reply:
@DarkFelas answered in another PR:
static allows to skip "this." pointer and the method is easier to discover by the compiler (?)
can't say for sure, but that's the explanation I managed to find.

Are there any downsides using static methods?

Reply:
Static modifier was removed.